app/game_manager.py

```diff
@@ -22,23 +26,217 @@ def __init__(self):
+        """Prints the game-menu screen."""
+
+        def print_options() -> None:
+            for i, option in enumerate(options):  # updates the options
+                if i == self.curr_highlight and i != 3:
+                    print(
+                        self.term.move_x(term_positions[i])
+                        + self.term.bold_white_on_green
+                        + str(option)
+                        + self.term.normal
+                        + self.term.move_x(0),
+                        end="",
+                    )
+                elif i == self.curr_highlight and i == 3:
+                    print(
+                        self.term.move_x(term_positions[i])
+                        + self.term.bold_white_on_red
+                        + str(option)
+                        + self.term.normal
+                        + self.term.move_x(0),
+                        end="",
+                    )
+                elif i == 3:
+                    print(
+                        self.term.move_x(term_positions[i])
+                        + self.term.bold_red
+                        + str(option)
+                        + self.term.normal
+                        + self.term.move_x(0),
+                        end="",
+                    )
+                else:
+                    print(
+                        self.term.move_x(term_positions[i])
+                        + self.term.bold_green
+                        + str(option)
+                        + self.term.move_x(0),
+                        end="",
+                    )
+            if self.curr_highlight != 9:
+                print(
+                    self.term.move_down(3)
+                    + self.term.green
+                    + self.term.center(brief[self.curr_highlight])
+                    + self.term.move_x(0)
+                    + "\n\n"
+                    + self.term.white
+                    + self.term.center("Press [ENTER] to confirm")
+                    + self.term.move_up(5),
+                    end="",
+                )
+            else:
+                print(
+                    self.term.move_down(3)
+                    + self.term.white
+                    + self.term.center("Press [TAB] for option selection")
+                    + self.term.normal
+                    + self.term.move_up(4)
+                    + self.term.move_x(0)
+                )
+
+        def select_option() -> None:  # updates the highlighter variable
+            if self.curr_highlight < 3:
+                self.curr_highlight += 1
+            else:
+                self.curr_highlight = 0
+
+        w, h = self.term.width, self.term.height
+        options = [" Create Game ", " Join Game ", " Settings ", " Exit "]
+        brief = [
+            "Creates an new game and waits for an opponent to join",
+            "Join a pre-existing game of your choice",
+            "Change game settings",
+            "Exit the game",
+        ]
```

Comment:
Move this to a constant and use dictionaries instead mapping the option to its brief description.

Reply:
Mentioned in d3b9115f79a78d035b60b0e98a57829c3cab09cb.